digitalfabrik/integreat-cms
# Automatically remove user accounts that remain unactivated for 7 days

### Short Description

This pull request introduces a new management command designed to delete user accounts that have not been activated for over a week. It also includes a cron job that executes this command every hour to ensure regular checks of user accounts.

### Proposed Changes
- Implement a management command to delete inactive user accounts older than one week.
- Set up a cron job that runs every hour to invoke the management command.
- Configure the `django_cron` app.

### Side Effects

- This feature may result in the deletion of a large number of inactive accounts upon release.

### Testing

To test this feature, follow these steps:
- Create a new user account.
- Access the Django admin panel and set the account's creation date to one week ago.
- Run the cron job using the command: `./tools/integreat-cms-cli runcrons "integreat_cms.core.jobs.inactive_user_cleanup_job.InactiveUserCleanupJob" --force`.
- Verify that the user account has been deleted.

### Resolved Issues

Fixes: #2076

__________________________________________________

[Pull Request Review Guidelines](https://digitalfabrik.github.io/integreat-cms/pull-request-review-guide.html)
